src/anemoi/training/train/train.py

```diff
@@ -62,9 +64,14 @@ def __init__(self, config: DictConfig) -> None:
         OmegaConf.resolve(config)
         self.config = config
 
-        # Default to not warm-starting from a checkpoint
+        # Set Transfer Learning based on the other if not provided
+        if self.config.training.transfer_learning is None:
+            self.config.training.transfer_learning = (
```

Comment:
Why is the config value being overwritten here?

Reply:
Tries to fill the value for old configs that have not been updated.
Loading a checkpoint using transfer learning on the same model and when not loading the optimizer state (self.config.training.load_weights_only=True) will actually not change the checpoint itself. Transfer learning becomes normal learning then.

The falg is actually not really necessary but it adds clarity.